### ISSUE TITLE
Plans for Tesseract in 2026

### ISSUE THREAD

**Issue by amitdo** [tesseract-ocr/tesseract#4498]
New Year, New Plans.

**Comment by rubyFeedback** [comment]
One thing that would be neat is a short intro style, "how to set up tesseract correctly".

I had great results in the past via english (that is, OCR on books .pdf written in english,
via the english training data of tesseract); but for the german language, I failed; none of
the umlauts were recognied for instance. Now I need this again because of legacy .pdf files
that I need to annotate or find the text. So I am researching again how to do this in
tesseract. Google search has really gotten so bad lately that a tutorial focusing on small
steps, would be nice(r).

**Comment by amitdo** [comment]
@stweil,


Any chance you'll be able to make a new release in July?

</br></br></br>
July **2026**... :-)

**Comment by stweil** [comment]
Yes, indeed, it is time for a new release, ideally with dropout (#4554). The test of a training with dropout is still open and my reason for the delay.

**Comment by stweil** [comment]
I plan to create a new release 5.5.3 in the next days (after #4581 was merged). Please report anything which is still missing for the release in the main branch.

**Comment by stweil** [comment]
[Release 5.5.3](https://github.com/tesseract-ocr/tesseract/releases/tag/5.5.3) is now available. Thank you to all contributors who reported issues and sent or reviewed pull requests!

**Comment by amitdo** [comment]
https://github.com/tesseract-ocr/tessdoc/blob/main/ReleaseNotes.md#v100

Tesseract version 1.00 was released 20 years ago!
